utils/rpms/daos.spec

```diff
@@ -145,6 +147,7 @@ Requires: %{name}-client = %{version}-%{release}
 Requires: python-pathlib
 %if (0%{?suse_version} >= 1315)
 Requires: libpsm_infinipath1
+Requires: librdmacm
```

Comment:
Hrm.  I guess because our tests will force verbs if it finds an `ib*` interface, we should do this.

But perhaps, alternatively we also should think about adding `daos-{client,server}-verbs` sub-packages that have this `Requires:` as well as a `Requires: daos-{client,server}` so that people don't have to guess what else they need to install to get verbs support.  Then the above `Requires: librdmacm` goes away and line 140 becomes:

```
Requires: %{name}-client-verbs = %{version}-%{release}
```

Come talk to me if you need more detail about this.

Reply:
NOTE: Based on the DAOS quality WG meeting, I will be reverting the forced usage of ofi+verbs once we prove ofi+verbs is passing all tests. CORCI-838 is created for address setting of CRT_PHY_ADDR_STR to of+verbs for weekly testing.